docs/topics/gradle.md

```diff
@@ -320,6 +298,49 @@ kotlin{
 >
 {type="note"}
 
+You can use the `kotlin-test` dependency in any shared or platform-specific source set as well.
+
+For Kotlin/JVM Gradle uses JUnit 4 by default; therefore, the `kotlin("test")` dependency resolves to the variant for 
+JUnit 4, namely `kotlin-test-junit`.
+
+You can choose JUnit 5 or TestNG by calling 
+[`useJUnitPlatform()`]( https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useJUnitPlatform) 
+or [`useTestNG()`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useTestNG) in the 
+test task of your build script.
+The following example is for a JVM project:
+
+<tabs>
+
+```groovy
+dependencies {
+    testImplementation 'org.jetbrains.kotlin:kotlin-test'
+}
+
+test {
+    useTestNG()
+}
+```
+
+```kotlin
+dependencies {
+    testImplementation(kotlin("test"))
+}
+
+tasks {
+    test {
+        useTestNG()
+    }
+}
+```
+
+</tabs>
+
+[Learn how to test code using JUnit on the JVM](jvm-test-using-junit.md).
+
+If you need to use a different JVM test framework, you can disable the automatic selection of a testing framework by 
+adding the line `kotlin.test.infer.jvm.variant=false` to the project’s `gradle.properties` and add this framework as a 
+dependency to your build script.
```

Comment:
This sentence is indeed too long. Let's think about splitting it.
Just an idea:
```suggestion
If you need to use a different JVM test framework, disable the automatic selection of a testing framework by 
adding the line `kotlin.test.infer.jvm.variant=false` to the project’s `gradle.properties`. Once you do this, add this this framework as a Gradle dependency.
```